debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/AbstractOracleDatatypesTest.java

```diff
@@ -99,16 +110,27 @@
             new SchemaAndValueField("VAL_BF", Schema.OPTIONAL_FLOAT32_SCHEMA, 1.1f),
             new SchemaAndValueField("VAL_BD", Schema.OPTIONAL_FLOAT64_SCHEMA, 2.22),
             new SchemaAndValueField("VAL_F", VariableScaleDecimal.builder().optional().build(), VariableScaleDecimal.fromLogical(VariableScaleDecimal.builder().optional().build(), new SpecialValueDecimal(new BigDecimal("3.33")))),
+            new SchemaAndValueField("VAL_F_10", VariableScaleDecimal.builder().optional().build(), VariableScaleDecimal.fromLogical(VariableScaleDecimal.builder().optional().build(), new SpecialValueDecimal(new BigDecimal("8.888")))),
             new SchemaAndValueField("VAL_NUM", Decimal.builder(6).parameter(PRECISION_PARAMETER_KEY, "10").optional().build(), new BigDecimal("4.4444")),
             new SchemaAndValueField("VAL_DP", VariableScaleDecimal.builder().optional().build(), VariableScaleDecimal.fromLogical(VariableScaleDecimal.builder().optional().build(), new SpecialValueDecimal(new BigDecimal("5.555")))),
             new SchemaAndValueField("VAL_R", VariableScaleDecimal.builder().optional().build(), VariableScaleDecimal.fromLogical(VariableScaleDecimal.builder().optional().build(), new SpecialValueDecimal(new BigDecimal("6.66")))),
+            new SchemaAndValueField("VAL_DECIMAL", Decimal.builder(6).parameter(PRECISION_PARAMETER_KEY, "10").optional().build(), new BigDecimal("1234.567891")),
+            new SchemaAndValueField("VAL_NUMERIC", Decimal.builder(6).parameter(PRECISION_PARAMETER_KEY, "10").optional().build(), new BigDecimal("1234.567891")),
             new SchemaAndValueField("VAL_NUM_VS", VariableScaleDecimal.builder().optional().build(), VariableScaleDecimal.fromLogical(VariableScaleDecimal.builder().optional().build(), new SpecialValueDecimal(new BigDecimal("77.323"))))
     );
 
     private static final List<SchemaAndValueField> EXPECTED_INT = Arrays.asList(
-            new SchemaAndValueField("VAL_INT", INTEGER_SCHEMA, new BigDecimal("1")),
-            new SchemaAndValueField("VAL_INTEGER", INTEGER_SCHEMA, new BigDecimal("22")),
-            new SchemaAndValueField("VAL_SMALLINT", INTEGER_SCHEMA, new BigDecimal("333"))
+            new SchemaAndValueField("VAL_INT", NUMBER_SCHEMA, new BigDecimal("1")),
```

Comment:
@gunnarmorling Now it feels very strange that we have subset of `NUMERIC` values represented as integer datattypes and integer datatypes defined at the database level are represented as `BigDecimal`. Would not it make it sense to introduce `BigInteger` logical datatype to cover such types? I know that we do the same for extra large types in MySQL but maybe this should be the correct solution from the beginning.

Reply:
My thinking is that people generally prefer the built-in KC types over custom ones. In that light, such `BigInteger` type wouldn't be really an advantage over using KC's `DECIMAL` with a scale of 0. Also we'll add a decimal handling mode of `string` which would be an alternative, simpler-to-consume representation. Does that make sense?

Reply:
For sure worth filing a JIRA issue to spark a discussion and if there's interest, we could send a PR.

Reply:
https://issues.apache.org/jira/browse/KAFKA-7199